modules/planning/on_lane_planning.cc

```diff
@@ -686,5 +687,64 @@ void OnLanePlanning::AddOpenSpaceOptimizerResult(
   (*warm_start_properties)["showLine"] = "true";
 }
 
+void OnLanePlanning::AddPartitionedTrajectory(
+    const planning_internal::Debug& debug_info,
+    planning_internal::Debug* debug_chart) {
+  // if open space info provider success run
+  if (!frame_->open_space_info().open_space_provider_success()) {
+    return;
+  }
+  // if empty return
+  auto chart = debug_chart->mutable_planning_data()->add_chart();
+  auto open_space_debug = debug_info.planning_data().open_space();
+
+  if (open_space_debug.chosen_trajectory().trajectory().empty() ||
+      open_space_debug.chosen_trajectory().trajectory()[0].
```

Comment:
Save `open_space_debug.chosen_trajectory().trajectory()` to variable to shorten code. DRY.